packages/EasyDoctrine/tests/Subscribers/EntityEventSubscriberTest.php

```diff
@@ -315,6 +315,95 @@ public function testEventsAreNotDispatchedWhenDispatcherIsDisabled(): void
         self::assertCount(0, $events);
     }
 
+    public function testEventsAreNotDispatchedWhenDispatcherIsDisabledForObject(): void
+    {
+        $eventDispatcher = new EventDispatcherStub();
+        $dispatcher = new DeferredEntityEventDispatcher($eventDispatcher);
+        $entityManager = EntityManagerStub::createFromDeferredEntityEventDispatcher(
+            $dispatcher,
+            [Category::class, Product::class],
+            [Category::class, Product::class]
+        );
+
+        $dispatcher->disable([Product::class]);
```

Comment:
We need to add examples for the new feature to be added into `docs/doctrine/entity_manager_decorator.md` (to the end).

Reply:
Done